hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1401,6 +1401,22 @@
    */
   public static final int BATCH_ROWS_THRESHOLD_DEFAULT = 5000;
 
+  public static final int DEFAULT_SLOW_LOG_RING_BUFFER_SIZE = 256;
```

Comment:
why do we still introduce these parameters in HConstants. I thought we have avoided to add parameters in it for quite a long time.

Reply:
That's right, we don't do this anymore. I think the initial patch had multiple references in various classes and hence, we kept all here.  e.g `SLOW_LOG_BUFFER_ENABLED_KEY` is being referred to at 4 places and similarly others too, but since this is a group of constant (usually something that should stay together), they are kept here.
Sounds good?